src/mito2/src/manifest/manager.rs

```diff
@@ -670,6 +593,6 @@ mod test {
 
         // get manifest size again
         let manifest_size = manager.manifest_usage();
-        assert_eq!(manifest_size, 1312);
+        assert_eq!(manifest_size, 1173);
```

Comment:
This test changes because the old checkpoint way is one less incremental file than the current region manifest. Originally, the incremental files are scanned by `self.store.scan(start_version, end_version).await`, left close right open range. And the `env_version` is `self.last_version`, which is already in current manifest. 

Same to the other tests below.